client/src/App.js

```diff
@@ -0,0 +1,39 @@
+import { Cart } from "./Components/private/Cart";
+import { ProductListing } from "./Components/ProductListingPage/ProductListing";
+import { WishList } from "./Components/private/WishList";
+import "./App.css";
+import { ProductPage } from "./Components/ProductPage/ProductPage";
+
+import { Navigation } from "./Components/nav";
+import { Homepage } from "./Components/Homepage/Homepage";
+import { Routes, Route } from "react-router";
+import { Login } from "./Auth/Login";
+import { PrivateRoute } from "./Components/private/PrivateRoute";
+import SignUpPage from "./Auth/SignUpPage";
+
+export default function App() {
+  return (
+    <div className='App'>
+      <Navigation />
+      <Routes>
+        <Route path='/'>
+          <Homepage />
+        </Route>
+        <Route path='/products/product/:productname/:productId'>
```

Comment:
the route name can be shorter?.....as the routes should be as short as possible for the user readibility?...just a suggestion though!

Reply:
yeah thanks for pointing it out, I have resolved it now